sdk/connectedvmwarevsphere/Azure.ResourceManager.ConnectedVMwarevSphere/tests/tests/Tests/ClusterTests.cs

```diff
@@ -0,0 +1,162 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using Azure.ResourceManager.Resources;
+using Azure.ResourceManager.Resources.Models;
+using Azure.ResourceManager.TestFramework;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+using Azure.Core;
+using System.Linq;
+using Azure.ResourceManager.ConnectedVMwarevSphere.Models;
+using Azure.ResourceManager.ConnectedVMwarevSphere.Tests.Helpers;
+
+namespace Azure.ResourceManager.ConnectedVMwarevSphere.Tests.tests.Tests
+{
+    public class ClusterTests : ConnectedVMwareTestBase
+    {
+        public ClusterTests(bool isAsync) : base(isAsync)
+        {
+        }
+
+        private async Task<VMwareClusterCollection> GetVMwareClusterCollectionAsync()
+        {
+            var resourceGroup = await CreateResourceGroupAsync();
+            return resourceGroup.GetVMwareClusters();
+        }
+
+        [AsyncOnly]
```

Comment:
Usually we do not add this attribute. Do we have a specific reason of using this attribute?

Reply:
I have removed this attribute and run test for sync and aysnc both.